### PR TITLE
updates wording

### PR DESCRIPTION
**Pull request by SvenMuller95**

### CODE REVIEW

File: HyperIndex_versioned_docs/version-v1/contract-import.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: contract-import
-title: No-Code Quickstart
-sidebar_label: No-Code Quickstart
+title: Quickstart
+sidebar_label: Quickstart
```
Comment (moose-code) [inline HyperIndex_versioned_docs/version-v1/contract-import.md:4]: @SvenMuller95 we need to make sure in the remappings that this is correctly redirected too

File: HyperIndex_versioned_docs/version-v1/supported-networks.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+id: supported-networks
+title: Supported Networks
+sidebar_label: Supported Networks
+slug: /supported-networks
+---
+
+HyperIndex natively supports indexing any EVM blockchain out of the box. As a developer you can start indexing and querying your smart contract data across any EVM-compatible L1, L2, or L3 blockchain using HyperIndex.
```
Comment (moose-code) [inline HyperIndex_versioned_docs/version-v1/supported-networks.md:8]: Love this

File: docs/HyperIndex/contract-import.md [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
 
 To start, simply run:
 
 ```
 envio init
 ```
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/zkVlGgf5XAo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/JOiLUysZf-s" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (moose-code) [inline docs/HyperIndex/contract-import.md:22]: Don't want this video elsewhere maybe?

File: docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md [hunk 4]
```diff
@@ -29,14 +29,17 @@ Here is a table of the currently supported networks on HyperRPC and their respec
 | Bsc Testnet          | 97         | https://bsc-testnet.rpc.hypersync.xyz or https://97.rpc.hypersync.xyz               |                 |
 | C1 Milkomeda         | 2001       | https://c1-milkomeda.rpc.hypersync.xyz or https://2001.rpc.hypersync.xyz            |                 |
 | Celo                 | 42220      | https://celo.rpc.hypersync.xyz or https://42220.rpc.hypersync.xyz                   |                 |
+| Chiliz               | 8888       | https://chiliz.rpc.hypersync.xyz or https://8888.rpc.hypersync.xyz                  |                 |
```
Comment (moose-code) [inline docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:32]: Does this not need to be generated by the code?

File: docs/HyperSync/hypersync-curl-examples.md [hunk 5]
```diff
@@ -286,7 +286,7 @@ curl --request POST \
 
 #### Get All Successful/Failed Transactions for the Last 10 Blocks
 
-First, query the current block height using https://eth.hypersync.xyz/height, subtract 10 from it, and store the result in a variable.
+First, query the current block height using https://eth.HyperSync.xyz/height, subtract 10 from it, and store the result in a variable.
```
Comment (moose-code) [inline docs/HyperSync/hypersync-curl-examples.md:289]: We must leave this lowercase

**Review by moose-code — COMMENTED** [review]
Great!! Main point of feedback is create a redirect from the no-code to the new quickstart page - so we don't get broken links given the rename. 

Great stuff sven!!

File: HyperIndex_versioned_docs/version-v1/contract-import.md (hunk 1 above)
Comment (SvenMuller95) [reply HyperIndex_versioned_docs/version-v1/contract-import.md:4]: Hey @moose-code , I don't think this affects the slug, URL or need redirect. Neither the sidebar.js  It's merely title and sidebar label (as it appears in the docs).

File: docs/HyperIndex/contract-import.md (hunk 3 above)
Comment (SvenMuller95) [reply docs/HyperIndex/contract-import.md:22]: i didn't remove the videos, they still there, just cut those lines and pasted into a different position on the same page.

File: docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md (hunk 4 above)
Comment (SvenMuller95) [reply docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:32]: I didn't intend to change anything here on this page. might have made a space or something. it looks fine locally and vercel.

File: docs/HyperSync/hypersync-curl-examples.md (hunk 5 above)
Comment (SvenMuller95) [reply docs/HyperSync/hypersync-curl-examples.md:289]: Agreed

Comment (SvenMuller95) [reply docs/HyperSync/hypersync-curl-examples.md:289]: Fixed in commit

**Review by moose-code — APPROVED** [review]
lgtm! Nice!

**Pull request merged**